packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1643,6 +1672,13 @@ class InputDecorator extends StatefulWidget {
   /// How the text in the decoration should be aligned horizontally.
   final TextAlign textAlign;
 
+  /// {@template flutter.widgets.inputDecorator.textAlignVertical}
+  /// How the text should be aligned vertically.
```

Comment:
Best to explain this a little more carefully, since what we're talking about depends on how the textfield is decorated.